oojs.md

```diff
@@ -7,15 +7,25 @@ Define a `Playlist` class. It should have...
 - An `addSong` (Function) method that adds a song to the `songs` property
 
 ```js
-Your answer goes here
+class Playlist={
+  constructor(title="",songs=[]){
+    this.title=title;
+    this.songs=songs
+  }   
+  favorites:0;
```

Comment:
`favorites` needs to be initialized inside of the constructor. You cannot have key-value pairs floating outside of methods in a class.